staff/judith-gongora/pre-course/bingo/bingo.js

```diff
@@ -0,0 +1,224 @@
+//Bingo Proyecto Tema 3
+var numerosdados = [] //Variable global Array para ir guardando números que han salido ya
+var linea = false //Variable global para comprobar si hay linea y que no vuelva a darnosla
+var turnos = 0 //Variable global que suma los turnos
+var bingoCard = [] //Variable global array para generar los numeros (objetos)
+var primero = true
+
+function newCard(){
+  primero = true
+  numerosdados = [] //Instanciamos de nuevo para retear el Array
+  turnos = 0 //Cada vez que generamos un cartón reseteamos la variable turnos
```

Comment:
Suggestion: do not use differents lenguages for variable declarations.

Reply:
You're right. From now on I will do everything in English.